api/test/integration/test_rbac_black_experimental_endpoints.tavern.yaml

```
@@ -106,17 +106,20 @@ stages:
     response:
       status_code: 200
       json:
-        error: 0
+        error: 1
         data:
-          affected_items:
-            - '000'
-            - '002'
-            - '004'
-            - '006'
-            - '008'
-          failed_items: []
-          total_affected_items: 5
-          total_failed_items: 0
+          affected_items: []
+          failed_items:
+            - error:
+                code: 1760
+              id:
+                - '000'
+                - '002'
+                - '004'
+                - '006'
+                - '008'
+          total_affected_items: 0
+          total_failed_items: 5
 
   - name: Request a list of agents (Partially allowed, user aware)
```

This is no longer `Partially allowed` since now all items are failed. In addition, this name was wrong since in this test no agents are being requested, `DELETE` method is used. Update the name in all those related tests where it is needed. 

```suggestion
  - name: Try to delete syscheck information of a list of agents (Denied, agents are newer than expected). 
```

---

Solved in dcf223e8899966a01c587648d57a3205c3e82da1